src/Documentation/Grain-Versioning/Backward-compatibility-guidelines.md

```diff
@@ -47,6 +47,34 @@ public interface IMyGrain : IGrainWithIntegerKey
 }
 ```
 
+Example of what can happen if you just rename the parameter names, let's say
+that we have the two following interface version deployed in the cluster:
+``` cs
```

Comment:
Add `**NOTE**: you should not do this change in your code, as it's an example of a bad practice that leads to very bad side-effects.`